models/user_model.py

```diff
@@ -93,6 +94,10 @@ def delete_user(username):
         db.session.commit()
         return done
 
+    def update_last_login(self):
```

Comment:
Consider adding error handling around db.session.commit() in update_last_login() to handle potential database errors.